src/evaluate.cpp

```diff
@@ -724,6 +721,11 @@ namespace {
                     - 43 * almostUnwinnable
                     - 100 ;
 
+    // Give more importance to non-material score
+    score = (score * 2 - materialScore) / 2;
```

Comment:
@Rocky640 is the line above correct, just wanted to confirm as in the PR description it says, 
> Instead of computing the initiative bonus on the material score + dynamic score
compute it on (material score/2) + dynamic score

Also before refactoring, we had `dynamicScore` added explicitly to the score like so:
`score += dynScore + initiative(dynScore * 2 + score);`
Is that still being done. 
Just wanted to make sure that the refactored changes are equivalent to the original code that passed.

Reply:
a) Bench after refactoring (but before rebasing) was same as the one tested on fishtest.
b) Bench after rebasing had been added to pr description.
Everything is fine.